src/api/app/views/webui2/webui/package/_commit_item.html.haml

```diff
@@ -0,0 +1,36 @@
+- commit = @package.commit(rev.to_s)
+- if commit
+  - if commit['user'] == '_service'
+    %i.fas.fa-cog.fa-lg
+    automatic update by service
+  - else
+    - user = User.find_by_login(commit['user'])
+    - if user
+      - realname = user.realname
+      = user_image_tag(user)
+      = link_to(user_show_path(user)) do
+        = realname.empty? ? user.login : "#{realname} (#{user.login})"
```

Comment:
I would prefer to write this positive... `realname.present?`... We need a new Rubocop cop: https://github.com/rubocop-hq/rubocop/issues/6129 :bulb:

Reply:
you can drop the `.login` as `user.login` is the same as `user.to_s`